extensions/wikia/Email/EmailMobileBadges.class.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class EmailMobileBadges
+ *
+ * @package Email
+ */
+class EmailMobileBadges {
+
+	static private $badges = [
```

Comment:
I think WikiaTech is unhappy because `$badges` is accessed via the full class name below, e.g. `EmailMobileBadges::$badges`.  You could shorten this by using `self` instead, e.g. `self::$badges`.